input/pagecontent/datax.md

```diff
@@ -32,12 +32,12 @@ The following resources are used in all data exchange transactions:
 | Library       | CRMI Shareable Library                     | [CRMI Shareable Library]                             |
 | Measure       | CRMI Shareable Measure Profile           | [CRMI Shareable Measure]                             |
 | MeasureReport | DEQM Data Exchange MeasureReport Profile | [DEQM Data Exchange MeasureReport Profile] |
-| Organization  | QI Core Organization Profile             | [QI Core Organization]                     |
-| Patient       | QI Core Patient Profile                  | [QI Core Patient]                          |
-| Practitioner  | QI Core Practitioner Profile             | [QI Core Practitioner]                     |
+| Organization  | QI-Core Organization Profile             | [QI-Core Organization]                     |
```

Comment:
Same as above, these are profiles names, do not change QI Core to QI-Core. Change it to "QICore" (no space) to match their profiles names. E.g., QICore Patient.